notebooks/MAST/PanSTARRS/PS1_DR2_TAP/PS1_DR2_TAP.ipynb

```diff
@@ -199,8 +208,8 @@
     "\"\"\".format(ra,dec,radius)\n",
```

Comment:
Comment for line 188/197:
`Mast._resolve_object(objname)` -> `Mast.resolve_object(objname)`
(Astroquery changed the name of this function)

Reply:
Updated this. The non-TAP HSC notebook may need updated as well, @rlwastro wrote that one.

Reply:
I can check and do a quick update PR on that one if needed.